src/FeedsCleanup/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Web.Xdt" version="2.1.0" targetFramework="net452" />
+  <package id="NDesk.Options" version="0.2.1" targetFramework="net452" />
```

Comment:
Can we use our own CommandLineArgs package instead? I know it's a DNX source package but we can just copy the source here.